app/assets/javascripts/comment.js

```diff
@@ -1,5 +1,4 @@
-(function() {
-
+$(function() {
   $('.comment-form').each(function() {
```

Comment:
It turns out that all along, these eventHandlers weren't being attached on document.load!

Reply:
Great catch :tada: